nessie/externals/asc_athletes_api.py

```diff
@@ -32,8 +32,9 @@
 def get_asc_feed():
     response = _get_asc_feed_response()
     if not response or not hasattr(response, 'json'):
-        app.logger.error(f'Bad response from ASC Athletes API: {response}')
-        return None
+        error = f'ASC API unexpected response: {response}'
```

Comment:
Thanks, @johncrossman -- this was probably left over from an early `@stow` of the feed.